script.js

```diff
@@ -1,9 +1,129 @@
 // eslint-disable-next-line no-unused-vars
 function closeMenu() {
   document.getElementById('myButtons').style.display = 'none';
+  const blur = document.getElementById('blur');
+  blur.classList.toggle('active');
 }
 
 // eslint-disable-next-line no-unused-vars
 function openMenu() {
   document.getElementById('myButtons').style.display = 'block';
-}
\ No newline at end of file
+  const blur = document.getElementById('blur');
+  blur.classList.toggle('active');
+}
+
+const pName = ['Tonic', 'Multi-Post Stories', 'Facebook360', 'Uber Navigation'];
+const description = "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea";
+const technologies = ['Html', 'Css', 'Javascript'];
+const featuredimg = ['./images/div0-bgr.png', './images/div1-bgr.png', './images/div2-bgr.png', './images/div3-bgr.png'];
+const liveLink = ['https://mattgomb.github.io/Portfolio-project/'];
+const sourceLink = ['https://github.com/MattGomb/portfolio-project'];
+const infoArray = [
+  {
+    id: 0,
+    name: pName[0],
+    featuredimg: featuredimg[0],
+    description,
+    technologies,
+    liveLink: liveLink[0],
+    sourceLink: sourceLink[0],
+  },
+  {
+    id: 1,
+    name: pName[1],
+    featuredimg: featuredimg[1],
+    description,
+    technologies,
+    liveLink: liveLink[0],
+    sourceLink: sourceLink[0],
+  },
+  {
+    id: 2,
+    name: pName[2],
+    featuredimg: featuredimg[2],
+    description,
+    technologies,
+    liveLink: liveLink[0],
+    sourceLink: sourceLink[0],
+  },
+  {
+    id: 3,
+    name: pName[3],
+    featuredimg: featuredimg[3],
+    description,
+    technologies,
+    liveLink: liveLink[0],
+    sourceLink: sourceLink[0],
+  },
+];
+
+let counter = -1;
+document.addEventListener('DOMContentLoaded', () => {
+  // const btn = document.querySelectorAll('.btn');
```

Comment:
- [ ] **Unnecessary commented codes make your code look bad and difficult to read therefore**, they should be removed to make your codebase neat. 😎 This is among coding best practices. Please remove the above-commented lin of codes.

Reply:
Thank you, removed!